client/js/src/main/scala/zio/zmx/client/frontend/views/DiagramView.scala

```diff
@@ -46,7 +49,37 @@ object DiagramView {
               s"A diagram for ${cfg.title}"
             ),
             div(
-              chart.element()
+              cls := "flex",
+              chart.element(),
+              div(
+                cls := "w-1/5 p-3 ml-2",
+                form(
+                  onSubmit.preventDefault.mapTo(
+                    Command.UpdateDiagram(cfg.copy(title = zipVar.now()))
+                  ) --> Command.observer,
+                  p(
+                    cls := "flex py-5",
+                    label(cls := "w-1/3 text-gray-50 text-xl font-bold", "Title: "),
+                    input(
+                      cls := "w-2/3 rounded-xl px-3 text-gray-600",
```

Comment:
I like the rounded style of the text field. Perhaps you can also add it to the conection url text field in the MainView, so that we have a consistent L&F.

Reply:
done